examples/hypervisor_cluster_profile.rb

```diff
@@ -44,8 +44,15 @@
   action :update
 end
 
-#Delets the Cluster5 Hypervisor Cluster Profile.
+#Deletes the Cluster5 Hypervisor Cluster Profile.
```

Comment:
As a best practice, would advise to please put a space between the hash symbol and Delete.

Reply:
ok

Reply:
done